presto/presto.go

```diff
@@ -587,14 +587,14 @@ type stmtStage struct {
 	SubStages       []stmtStage `json:"subStages"`
 }
 
-// ErrQueryId indicates that a query is success, only for give QueryId.
-type ErrQueryId struct {
-	QueryId string
+// EOF indicates that a query is success, only for give QueryID.
```

Comment:
Please update the comment to reflect what the error means: EOF indicates the server has returned io.EOF for the given QueryID.

Reply:
This is a change of behaviour that might affect existing code currently checking for io.EOF from query responses.

Reply:
I have update the comment, and please check what else needs to be modified

Reply:
> This is a change of behaviour that might affect existing code currently checking for io.EOF from query responses.

sorry for can not find  better solution to solve the problem